### PR TITLE
Add Nushell config

### PR DESCRIPTION
**Pull request by ElNico56**
Added a tere function for nushell, all tests work, arguments pass correctly, and --version and --help don't raise any errors, :3

### CODE REVIEW

**Comment by mgunyho** [comment]
Cool, thanks! A few very small things:
- Let's put nushell between xonsh and powershell, so that the "unixy" shells are first and the "windowsy" shells afterwards
- Is there a specific version of nushell that users should have for this to work? Based on #94, the argument passing didn't work earlier this year. Could you please mention that version in the instructions (like how the Xonsh instruction says currently)?
- Please use 4 spaces instead of tabs to be consistent with the other examples

**Comment by ElNico56** [comment]
1. of course :3
2. yea i saw that, i don't think it's a version issue but they just forgot to add --wrapped
3. of course :3

**Comment by ElNico56** [comment]
Ok yea, looked thru the changelog and sure enough, 0.88.0 in december 2023 added the desired behavior to --wrapped, added version requirement note

File: README.md [hunk 1]
```diff
@@ -82,6 +82,21 @@ aliases["tere"] = _tere
 ```
 </details>
 
+<details>
+<summary>Nushell</summary>
+
+Put this in your `config.nu` (Nusbell 0.88.0 or newer is required):
```
Comment (mgunyho) [inline README.md:88]:
```suggestion
Put this in your `config.nu` (Nushell 0.88.0 or newer is required):
```

**Pull request merged**